src/NzbDrone.Core/Indexers/Definitions/Anidub.cs

```diff
@@ -0,0 +1,542 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using FluentValidation;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Exceptions;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Messaging.Events;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Indexers.Definitions
+{
+    public class Anidub : TorrentIndexerBase<AnidubSettings>
+    {
+        public override string Name => "Anidub";
+        public override string[] IndexerUrls => new string[] { "https://tr.anidub.com/" };
+        public override string Description => "Anidub is russian anime voiceover group and eponymous anime tracker.";
+        public override string Language => "ru-ru";
+        public override Encoding Encoding => Encoding.UTF8;
+        public override DownloadProtocol Protocol => DownloadProtocol.Torrent;
+        public override IndexerPrivacy Privacy => IndexerPrivacy.Public;
```

Comment:
Given this requires auth is it really public?

Reply:
Nice catch, Copy-paste error. This tracker is semi-public. Fixed that.